src/data/community-events.json

```
@@ -26,6 +26,15 @@
     "startDate": "2023-03-16",
     "endDate": "2023-03-18"
   },
+  {
+    "title": "Ethereum Zürich",
+    "to": "https://ethereumzuri.ch/",
+    "sponsor": null,
+    "location": "Zürich, Switzerland",
```

We should use the common English spelling here

```suggestion
    "location": "Zurich, Switzerland",
```